### ISSUE TITLE
installing 0.21.1 fails using documented install steps

### ISSUE THREAD

**Issue by jmazzitelli** [operator-framework/operator-lifecycle-manager#2776]
## Bug Report

I am following the install steps found here: https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.21.1 where it says to run:

```
curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.21.1/install.sh -o install.sh
chmod +x install.sh
./install.sh v0.21.1
```

I did that and I get the following error and OLM fails to install properly:

```
$ curl -L https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.21.1/install.sh -o install.sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  1884  100  1884    0     0   8448      0 --:--:-- --:--:-- --:--:--  8448
$ chmod +x install.sh
$ ./install.sh v0.21.1
customresourcedefinition.apiextensions.k8s.io/catalogsources.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/installplans.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/olmconfigs.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/operatorconditions.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/operatorgroups.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/operators.operators.coreos.com unchanged
customresourcedefinition.apiextensions.k8s.io/subscriptions.operators.coreos.com unchanged
The CustomResourceDefinition "clusterserviceversions.operators.coreos.com" is invalid: metadata.annotations: Too long: must have at most 262144 bytes
```

**Environment**

* Kubernetes version information:

```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.0", GitCommit:"cb303e613a121a29364f75cc67d3d580833a7479", GitTreeState:"clean", BuildDate:"2021-04-08T16:31:21Z", GoVersion:"go1.16.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"22", GitVersion:"v1.22.3", GitCommit:"c92036820499fedefec0f847e2054d824aea6cd1", GitTreeState:"clean", BuildDate:"2021-10-27T18:35:25Z", GoVersion:"go1.16.9", Compiler:"gc", Platform:"linux/amd64"}
```

* Kubernetes cluster kind: 

Minikube v1.24.0:

```
$ minikube version
minikube version: v1.24.0
commit: 76b94fb3c4e8ac5062daf70d60cf03ddcc0a741b
```

**Comment by timflannagan** [comment]
@jmazzitelli yep we just merged a fix to the install script in #2767. We'll need to cut a v0.21.2, or update that install bash script release artifact.

**Comment by jmazzitelli** [comment]
@timflannagan thanks! Is there an ETA on when 0.21.2 will be released? I am using this in a CI script (running tests on my operator when installed via OLM) so I would need to update that CI script that launches everything. I really don't want to implement a fix to update the OLM install script only to have to rip it out again when 0.21.2 is released - so I could just wait for 0.21.2 if it will be coming soon.

**Comment by timflannagan** [comment]
>  I really don't want to implement a fix to update the OLM install script only to have to rip it out again when 0.21.2 is released - so I could just wait for 0.21.2 if it will be coming soon.

Understandable - I think we'll cut release today so I don't think you need to make any short term workarounds.

**Comment by exdx** [comment]
@jmazzitelli the `install.sh` script has been updated for the v0.21.1 release payload, so you should not see that CI failure anymore.

**Comment by exdx** [comment]
Hi @jmazzitelli, this issue should be resolved by the amended script in v0.21.1. There was also a new v0.21.2 release which incorporates the new script in the payload. Please reopen if you have any additional issues.
